src-built-in/components/shared/style.css

```diff
@@ -62,20 +75,20 @@
 }
 
 .app-tag {
-	height: 17px;
-	border-radius: 2px;
+	height: 20px;
+	border-radius: 12px;
 	background-color: var(--tag-background-color);
 	user-select: none;
 	margin: 0 4px 4px 0;
-	padding: 4px 5px 0 5px;
+	padding: 5px 7px 0 9px;
```

Comment:
we need a "px" after "0"
(I think if we're setting all 4 directions, we need "px" after every single one of them, if we're using left-right top-down format, like ```padding: 0 30px```, we can leave out the "px" after "0")